kiwi/container/__init__.py

```diff
@@ -30,9 +30,9 @@ class ContainerImage(object):
     :param string name: container system name
     :param string root_dir: root directory path name
     """
-    def __new__(self, name, root_dir):
+    def __new__(self, name, root_dir, custom_args=None):
```

Comment:
ok done, see kiwi/builder/container.py, there all the data gets passed. I will now add the runtime check for umoci and skopeo so that we don't need to add them as requires in the spec file for the first shot